packages/core/src/browser/tree/tree-decorator.ts

```diff
@@ -416,8 +435,12 @@ export namespace TreeDecoration {
                     if (data.length > 0) {
                         result.push({ data });
                     }
-                    const { length } = ranges.splice(containerIndex, 1).shift()!;
-                    result.push({ data: caption.substr(i, length), highligh: true });
+                    const { length, color } = ranges.splice(containerIndex, 1).shift()!;
+                    if (color) {
+                        result.push({ data: caption.substr(i, length), color: color });
+                    } else {
+                        result.push({ data: caption.substr(i, length), highligh: true, color: color });
```

Comment:
We have to fix this typo: `highligh`.

Reply:
Not sure what the policy is in Theia about drive-by fixes.